### PR TITLE
Bookstore: styling

### PR DESCRIPTION
**Pull request by JuanDiegoGuerra**
### Project requirements - Bookstore

- Add any missing **JSX elements** to your project (without adding extra functionality) ✅
- Your final design should match **Zeplin's** design (this includes all elements + the same styling) ✅
- You may use the **CSS rules** detailed in **Zeplin** to style the website. ✅
- All **linters** are passing! ✅

### CODE REVIEW

File: src/components/AddForm.js [hunk 1]
```diff
@@ -23,27 +23,30 @@ const AddForm = () => {
...
+      <form className="add-form" onSubmit={handleAdd}>
+        <label className="grow-3" htmlFor="bookTitle">
+          <input className="input-control input-title" type="text" id="bookTitle" placeholder="Title" value={title} onChange={handleTitleChange} required />
+        </label>
+        <br />
+        <label className="grow-3" htmlFor="bookAuthor">
+          <input className="input-control input-author" type="text" id="bookAuthor" placeholder="Author" value={author} onChange={handleAuthorChange} required />
+        </label>
+        <br />
+        <button className="input-control input-send" type="submit">ADD BOOK</button>
```
Comment (TadesseDev) [inline src/components/AddForm.js]: - [ ] Dear @JuanDiegoGuerra, Your design is really great, but the add book form is a bit different from the original design. As you can see the original design has a `Category` form element which is a select field. Please make sure to follow the original design closely. 

### Your design 
Screen Shot 2023-08-25 at 5 58 44 PM

### Original design 
Screen Shot 2023-08-25 at 5 55 19 PM

File: src/components/Header.js [hunk 2]
```diff
@@ -1,12 +1,14 @@
...
 
 const Header = () => (
-  <header>
-    <h1>Bookstore CMS</h1>
-    <ul>
-      <li><NavLink to="/">BOOKS</NavLink></li>
-      <li><NavLink to="/categories">CATEGORIES</NavLink></li>
+  <header className="header-container">
+    <h1 className="header-title">Bookstore CMS</h1>
+    <ul className="header-nav">
```
Comment (TadesseDev) [inline src/components/Header.js:7]: - [ ] Dear @JuanDiegoGuerra, For the font family, I notice you have not applied any fonts, as you can see from the `zeplin` design, the main font for the heading is `Montserrat` please apply the font family. 

### Your design 
Screen Shot 2023-08-25 at 6 05 22 PM
### Original design 
Screen Shot 2023-08-25 at 6 06 17 PM

- [ ] Also, the original design is a bit smaller in font sizing, Please consider making some adjustments to the sizing also.

**Review by TadesseDev — CHANGES_REQUESTED** [review]
### Hi @JuanDiegoGuerra 

### I'm Tadesse Alemayehu your reviewer for this PR. :hand: 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
212094426-b41dd457-c9db-4ab3-bbe9-176dd8dd08b7




## Highlights
 - [x] Your design is almost identical to the original design, there are just some issues to be addressed ✅ 
 
## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

- N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by JuanDiegoGuerra** [comment]
###  Required changes made 🛠
**All linters are enable!**
- **Fixed** _"...As you can see the original design has a `Category` form element which is a select field. Please make sure to follow the original design closely."_ 🔨✅
- **Fixed** _"For the font family, I notice you have not applied any fonts, as you can see from the `zeplin` design, the main font for the heading is `Montserrat` please apply the font family."_ 🔨✅
- **Fixed** _"Also, the original design is a bit smaller in font sizing, Please consider making some adjustments to the sizing also."_ 🔨✅

image

**Review by Strangeal — APPROVED** [review]
Hi @JuanDiegoGuerra, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 




## Highlights
- Added `category` input ✅ 
- Added `font-family` ✅ 
- Added `font-size` ✅ 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**